WhatShouldITestTests/MenuTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WhatShouldITest.Entities;
+
+namespace WhatShouldITestTests
+{
+    [TestClass]
+    public class MenuTests
+    {
+        [TestMethod]
+        public void Special_AfterConstruction_ShouldReturnValue()
+        {
+            var menu = new Menu(null);
+
+            menu.DailySpecial.Should().NotBeNullOrEmpty();
+        }
```

Comment:
Here I've created a test that checks that the DailySpecial property returns something after construction. That's great, right? Our code coverage for the Menu class definitely went up. It's at 85% now!
![image](https://user-images.githubusercontent.com/668946/70744537-7e49a480-1ce7-11ea-920a-97be8dc7ea73.png)

But what are we actually testing?